test/measures/2018-measures-data-spec.js

```diff
@@ -1,162 +0,0 @@
-const chai = require('chai');
-const assert = chai.assert;
-const _ = require('lodash');
-const fs = require('fs');
-const path = require('path');
-const parse = require('csv-parse/lib/sync');
-
-const mipsDataFormat = require('../../index.js');
-const actualAciRelation = require('../../util/measures/aci-measure-relations.json');
-const actualCpcPlusGroups = require('../../util/measures/cpc+-measure-groups.json');
-const actualMeasureSpecificationData = parse(fs.readFileSync(path.join(__dirname, '../../util/measures/measurePDF-Specification.csv'), 'utf8'));
-
-const year = 2018;
-const measuresData = mipsDataFormat.getMeasuresData(year);
-describe(year + ' measures data json', function() {
-  const measureIds = _.map(measuresData, 'measureId');
-
-  it('should not have any duplicate measureIds', function() {
-    assert.equal(_.uniq(measureIds).length, measureIds.length);
-  });
-
-  describe('pre-aci attestations', function() {
```

Comment:
Advice: Instead of deleting all these tests, you could just change all the `it()`s to `xit()`, that way they're easy to turn back on once that data is imported

Reply:
i did do that at first, but then i realized that some (many?) tests would no longer be relevant, so it seemed cleaner just to delete. i left a comment at the top of the file telling people to go look in the 2017 file instead.